src/lib/Zikula/Composer/CopyRequirementsFile.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Zikula package.
+ *
+ * Copyright Zikula Foundation - http://zikula.org/
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Zikula\Composer;
+
+use Composer\Script\Event;
+
+/**
+ * A class to rewrite RequireJS configuration
+ */
+class CopyRequirementsFile
+{
+    /**
+     * This function generates from the customized bootstrap.less und font-awesome.less a combined css file
+     *
+     * @param string|null Where to dump the generated file
+     */
+    public static function copy(Event $event)
+    {
+        copy('src/vendor/sensio/distribution-bundle/Resources/skeleton/app/SymfonyRequirements.php', 'src/app/SymfonyRequirements.php');
```

Comment:
This shouldn't use relative paths. Maybe look at https://github.com/sensiolabs/SensioDistributionBundle/blob/master/Composer/ScriptHandler.php#L185-L196 to see how they do it.